css/style.css

```diff
@@ -104,14 +104,27 @@ input {
 	width: 100%;
 	max-width: 500px;
 	border: 2px solid #000000;
-	margin: 10px 10px 10px 10px;
+	margin: 10px 0 10px 0;
 	padding: 10px 10px 10px 10px;
 	text-align: center;
 	outline: none;
 	background-color: #ffffff;
 	transition: all 0.5s;
+	border-radius: 0;
+    -webkit-border-radius:0;
```

Comment:
Not setting a border-radius was causing mobile Safari to use the default rounded border styles.